.github/workflows/ci-nix.yml

```diff
@@ -444,7 +452,7 @@ jobs:
         run: |
           bins="${{ matrix.build.bins }}"
           for bin in ${bins//,/ } ; do
-            nix run "github:Mic92/nix-fast-build?rev=4376b8a33b217ee2f78ba3dcff01a3e464d13a46" -- --skip-cached --no-nom --flake .#legacyPackages.$(nix eval --raw --impure --expr builtins.currentSystem).$bin && sha256sum "./result/bin/$bin"
+            nix build .#$bin && sha256sum "./result/bin/$bin"
```

Comment:
Is this step meant to have a test similar to the check versions step above? I believe this only prints the sha256 hash without comparing.

```
bin=fedimint-cli
nix build .#$bin && sha256sum "./result/bin/$bin"
20e9140fd21367f8b53f6a26f2a75bcf7191b5124058722c3f991e3e4e1b8e58  ./result/bin/fedimint-cli
```

Reply:
It's only for printing, so one can take it from CI output and compare with a local build.